doc/man1/openssl-req.pod.in

```diff
@@ -148,7 +148,9 @@ Prints out the value of the modulus of the public key contained in the request.
 
 =item B<-verify>
 
-Verifies the self-signature on the request.
+Verifies the self-signature on the request. If the verification fails,
+the program will immediately exit, i.e. further option processing
+(e.g. B<-text>) is skipped.
```

Comment:
This should also be amended in `doc/man1/openssl-crl.pod.in`

Reply:
Thanks, done.